scripts/funcTests/runFuncTests.sh

```diff
@@ -73,6 +73,28 @@ do
 	fi
 done
 
+#run mono test
+TestDir="$(pwd)/artifacts/NuGet.CommandLine.Test/14.0/Release"
+XunitConsole="$(pwd)/packages/xunit.runner.console.2.1.0/tools/xunit.console.exe"
+NuGetExe="$(pwd)/.nuget/nuget.exe"
+
+#Get NuGet.exe
+wget -O $NuGetExe https://dist.nuget.org/win-x86-commandline/latest-prerelease/nuget.exe 
+
+#restore solution packages
+mono $NuGetExe restore  "$(pwd)/.nuget/packages.config" -SolutionDirectory "$(pwd)"
+if [ $? -ne 0 ]; then
+	echo "Restore failed!!"
+	exit 1
+fi
+
+#Clean System dll
+rm -r -f "$TestDir/System.*" "$TestDir/WindowsBase.dll" "$TestDir/Microsoft.CSharp.dll" "$TestDir/Microsoft.Build.Engine.dll"
+
+#Run xunit test
+
+mono $XunitConsole "$TestDir/NuGet.CommandLine.Test.dll"
```

Comment:
Can haz tracking item for other test projects we should run on mono?
e.g. NuGet.CommandLine.FuncTest